src/components/application_manager/rpc_plugins/sdl_rpc_plugin/src/commands/mobile/perform_interaction_request.cc

```diff
@@ -93,9 +93,9 @@ bool PerformInteractionRequest::Init() {
 
   if (mobile_apis::InteractionMode::BOTH == interaction_mode_ ||
       mobile_apis::InteractionMode::MANUAL_ONLY == interaction_mode_) {
-    default_timeout_ = default_timeout_ + (request_timeout_ * 2);
+    default_timeout_ += request_timeout * 2;
```

Comment:
@ValeriiMalkov may be is better to move value 2 to const variable with appropriate name, in order to avoid magic values?

Reply:
@ZhdanovP See please commit feedf047105e492833c264de6001f8bfdd50943e